pyart/util/datetime_utils.py

```diff
@@ -1,30 +1,62 @@
 """ Functions for converting date and time between various forms """
 
-from netCDF4 import num2date
+from netCDF4 import num2date, date2num
 
+EPOCH_UNITS = "seconds since 1970-01-01T00:00:00Z"
 
-def datetime_from_radar(radar):
+def datetime_from_radar(radar, epoch=False):
     """ Return a datetime for the first ray in a Radar. """
-    return num2date(radar.time['data'][0], radar.time['units'])
+    if epoch:
+        dtrad = num2date(radar.time['data'][0], radar.time['units'])
+        epnum = date2num(dtrad, EPOCH_UNITS)
+        return num2date(epnum, EPOCH_UNITS)
+    else:
+        return num2date(radar.time['data'][0], radar.time['units'])
 
 
-def datetimes_from_radar(radar):
+def datetimes_from_radar(radar, epoch=False):
     """ Return an array of datetimes for the rays in a Radar. """
-    return num2date(radar.time['data'][:], radar.time['units'])
+    if epoch:
+        dtrad = num2date(radar.time['data'][:], radar.time['units'])
+        epnum = date2num(dtrad, EPOCH_UNITS)
+        return num2date(epnum, EPOCH_UNITS)
+    else:
+        return num2date(radar.time['data'][:], radar.time['units'])
 
 
-def datetime_from_dataset(dataset):
+def datetime_from_dataset(dataset, epoch=False):
     """ Return a datetime for the first time in a netCDF Dataset. """
-    return num2date(dataset.variables['time'][0],
+    if epoch:
+        dtdata = num2date(dataset.variables['time'][0],
+                         dataset.variables['time'].units)
```

Comment:
This line should have an extra space to align with the dataset in the line above to meet PEP8 style guidelines.